src/dialogs/PaymentErrorDialog.vue

```diff
@@ -103,5 +103,21 @@ export default class PaymentErrorDialog extends Vue {
   get numWarnings (): number {
     return this.warnings?.length || 0
   }
+
+  /** The formated error message */
+  protected formatedError = []
+  protected mounted (): void {
+    this.formatedError = this.errors.map((item:any) => (
+      item.message.split('<br/>').map((item2:string, index2:number) => {
+        return index2 === 0 ? item2 + '<br/>' : '<span>' + item2 + '</span><br/>'
+      }).join('')
+    ))
+  }
 }
 </script>
+
+<style scoped>
+  #formated-error-msg >>> span {
```

Comment:
The deep CSS selector should not be needed as the span is not rendered inside a Vue component.